docs/content/direct/core-chart.md

```diff
@@ -190,89 +190,115 @@ the `kflex` CLI and one not.
+      -n $(kubectl get controlplane $cpname -o=jsonpath='{.status.secretRef.namespace}') \
+      -o=jsonpath="{.data.$(kubectl get controlplane $cpname -o=jsonpath='{.status.secretRef.key}')}" \
+      | base64 -d > "kubeconfig-$cpname"
+  fi
+  curname=$(kubectl --kubeconfig "kubeconfig-$cpname" config current-context)
+  if [ "$curname" != "$cpname" ]
+  then kubectl --kubeconfig "kubeconfig-$cpname" config rename-context "$curname" $cpname
+  fi
+done
+```
+
+The code above puts the kubeconfig for a control plane `$cpname` into a file name `kubeconfig-$cpname` in the local folder.
+The current context will be renamed to `$cpname`, if it does not already have that name (which it will for control planes of type "k8s", for example).
+
+With the above kubeconfig files in place, the control plane named `$cpname` can be accessed as follows.
+
+```shell
+kubectl --kubeconfig "kubeconfig-$cpname" ...
+```
+
+The individual kubeconfigs can also be merged as contexts of the current `~/.kube/config` with the following commands:
+
+```shell
+echo "Merging the Control Planes kubeconfigs into ~/.kube/config ..."
+cp ~/.kube/config ~/.kube/config.bak
+KUBECONFIG=~/.kube/config:$(find . -maxdepth 1 -type f -name 'kubeconfig-*' | tr '\n' ':') kubectl config view --flatten > ~/.kube/kubeconfig-merged
+mv ~/.kube/kubeconfig-merged ~/.kube/config
+```
+
+Afterwards the content of a KubeFlex Control Plane `$cpname` can be accessed by specifing its context:
+
+```shell
+kubectl --context "$cpname" ...
+```
+
+3. using `import-cp-contexts.sh` script
+
+The following covenience command can also be used to import all the KubeFlex Control Planes in the current hosting cluster as contexts of the current kubeconfig:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v$KUBESTELLAR_VERSION/scripts/import-cp-contexts.sh) --merge
+```
+
+The script above only requires `kubectl` and `yq`.
+
+The script accepts the following arguments:
+
+- `--kubeconfig <filename>` specify the kubeconfig of the hosting cluster where the KubeFlex Control Planes are located. Note that this argument will override the content of the `KUBECONFIG` environment variable
+- `--context <name>` specify a context of the current kuneconfig where to look for KubeFlex Control Planes. If this argument is not specified, then all contexts will be searched.
+- `--names|-n <name1>,<name2>,..` comma separated list of KubeFlex Control Planes names to import. If this argument is not specified then *all* available KubeFlex Control Planes will be imported.
+- `--replace-localhost|-r <address>` replaces server addresses "127.0.0.1" with a desired `<address>`. This parament is useful for making KubeFlex Control Planes of type `host` accessible from outside the machine hosting the cluster.
+- `--merge|-m` merge the control planes contexts with the existing cluster contexts. If this flag is not specified, then only the contexts of the KubeFlex Control Planes will be returned.

```

Comment:
1. English grammar: make it "control plane contexts" instead of "control planes contexts".
2. "will be returned" is not always right: depending on `-o`, this script will do a side-effect rather than a "return". I think that something like "will be produced" could cover all the cases.

Reply:
ok